types/styled-system/styled-system-tests.tsx

```diff
@@ -137,8 +139,8 @@ const breakpointsPx = breakpoints.map(px);
 const mediaQueries = breakpoints.map(createMediaQuery);
 
 const boxStyle = variant({
-    prop: 'boxStyle',
-    key: 'box',
+    prop: "boxStyle",
```

Comment:
hm, looks like `prettier` was set to format on save globally. I can revert this, but changes are minimal and this does fix the split tab/spaces and single/double quotes within the file. Reviewer's choice.